demo/api/src/db/fixtures/fixtures.console.ts

```diff
@@ -41,6 +41,7 @@ const getDefaultPageInput = (): PageInput => {
 @Console()
 export class FixturesConsole {
     constructor(
+        // @ts-expect-error Updating to nestjs/common 10 will fix this error (see: https://github.com/nestjs/mongoose/issues/1717#issuecomment-1473733362 and https://github.com/microsoft/TypeScript/issues/52435)
```

Comment:
I can add the same line to the FixturesConsole in the Starter without problems... Could you please investigate if there are any differences in the TSConfig that could be responsible for this?